cla_backend/apps/call_centre/management/commands/cla_superuser_organisation_perms.py

```diff
@@ -0,0 +1,19 @@
+from django.core.management import BaseCommand
+from django.contrib.auth.models import Group, Permission
+
+
+class Command(BaseCommand):
+    help = "Assign CLA Superusers 'Can add organisation' and 'Can change organisation' permissions"
+
+    def handle(self, *args, **options):
+        group = Group.objects.get(name="CLA Superusers")
+        if not group:
+            return
+
+        can_add_organisation_perm = Permission.objects.get(name="Can add organisation")
+        if can_add_organisation_perm:
```

Comment:
As above with regard to use of `.get()`

In this case, though, it might be preferable to just use a filter and pass the result (whether there's anything in it or not) to the `.add()` without checking.

Something like -- _and I haven't tested this_ -- ```group.permissions.add(Permission.objects.filter(name="Can add organisation"))```

Or even combining the two steps
```group.permissions.add(Permission.objects.filter(name__in=["Can add organisation", "Can change organisation"]))```
again, not tested at all :)

Reply:
`group.permissions.add(Permission.objects.filter(name__in=["Can add organisation", "Can change organisation"]))`

Didn't work for me as group.permissions.add wasn't expecting queryset.

However 
`permissions = Permission.objects.filter(name__in=["Can add organisation", "Can change organisation"])
groups.first().permissions.add(*permissions)
`
has worked for me